docs/how-to-guides/deploy-feathr-api-as-webapp.md

```diff
@@ -0,0 +1,108 @@
+# Feathr API
+The API currently supports following functionality
+
+1. Get Feature by Qualified Name
+2. Get Feature by GUID
+3. Get List of Features
+4. Get Lineage for a Feature
+
+
+## Build and run locally
+### Install
+
+You can install dependencies through the requirements file
+```bash
+$ pip install -r requirements.txt
+```
+
+### Run
+This command will start the uvicorn server locally and will dynamically load your changes.
+```bash
+uvicorn api:app --port 8080 --reload
+```
+
+## Build and deploy on Azure
+Here are the steps to build the API as a docker container, push it to Azure Container registry and then deploy it as webapp. The instructions below are for Mac/Linux but should work on Windows too. You might have to use sudo command or run docker as administrator on windows if you don't have right privileges.
+
+1. Install Azure CLI by following instructions [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+1. Create Azure Container Registry. First create the resource group.
```

Comment:
Why are these all 1. xxx , and 1.xxx?

Reply:
Not sure if you are asking why am I using 1. for numbering and not 1. 2. 3. etc? This is an easy way for ordered list numbering in GH markdown so if I delete an entry or add one in between I don't have to readjust all the other numbers.